src/Sulu/Bundle/AdminBundle/Resources/js/components/Url/tests/Url.test.js

```diff
@@ -149,10 +149,10 @@ test('Call onChange callback with undefined when path was changed to invalid url
         },
     });
 
-    expect(changeSpy).toBeCalledWith(undefined);
+    expect(changeSpy).toBeCalledWith('https://sulu.a');
 });
 
-test('Call onChange callback with undefined if URL is not valid but leave the current value', () => {
+test('Call onChange callback if url is not valid but leave the current value', () => {
```

Comment:
does this test make any sense as the url is I think from our case of view valid, or do I missunderstand and there is still a validation done?

Reply:
I just want to test, that the onChange callback is now be called with the actual url instead of `undefined` like it was before.